### PR TITLE
Added staff of the dead special attack to the timers plugin

### PR DESCRIPTION
**Pull request by rune3132**
Also works for weapons with similar special attacks like the staff of light

### CODE REVIEW

**Comment by deathbeam** [comment]
You should probably fix your formatting here

**Comment by deathbeam** [comment]
Maybe [Magic Fang](http://oldschoolrunescape.wikia.com/wiki/Magic_fang) would be perfect

File: runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersConfig.java [hunk 1]
```diff
@@ -244,4 +244,13 @@ default boolean showSuperantipoison()
 	{
 		return true;
 	}
+	@ConfigItem(
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersConfig.java]: Add new line above

File: runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersConfig.java [hunk 2]
```diff
@@ -244,4 +244,13 @@ default boolean showSuperantipoison()
...
+			keyName = "showSoTDspec",
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersConfig.java]: Fix your indentation level

File: runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java [hunk 3]
```diff
@@ -303,6 +309,14 @@ public void onChatMessage(ChatMessage event)
 		{
 			createGameTimer(EXSUPERANTIFIRE);
 		}
+		if (config.showSoTDspec() && event.getMessage().equals("<col=007f00>Spirits of deceased evildoers offer you their protection.</col>"))
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java]: Add new lines before and after if statements

**Comment by zeruth** [comment]
Just saying most IDE's have a source format option, and code cleanup. You should use them before posting pulls.

**Pull request closed without merging**